include/list.h

```diff
@@ -0,0 +1,241 @@
+#ifndef _LLIST_H
```

Comment:
Is this the same file as before? If so, you can't just delete the license and be done with it, we cannot incorporate GPL code. Can't you use our queue.h?